source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+---
+title: Miele
+description: Instructions on how to set up the Miele integration within Home Assistant.
+ha_category:
+  - Hub
+  - Sensor
+ha_iot_class: Cloud Push
+ha_release: '2025.5.0'
+ha_domain: miele
+ha_codeowners:
+  - '@astrandb'
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Miele integration allows users to integrate their home appliances using the [official 3rd party API](https://www.miele.com/developer). The integration will make the best effort to map the data-points in the API to sensors, switches, etc. in Home Assistant.
+
+## Use cases
+
+- Monitor the multiple sensors of the appliance and trigger automations based on these sensors.
+- Monitor the program status of the appliances.
+
+{% note %}
+Note that the feature availability depends on the appliance model.
+{% endnote %}
+
+## Supported devices
+
+You can find information about supported devices on the [Miele website](https://www.miele.com/developer/capabilities.html).
```

Comment:
I think its a bit more complicated, because we also need to support the device in HA

Reply:
Well all known device types has a status sensor so they are supported in that sense. When new platforms and new entity descriptions are added the coverage will grow gradually.

Reply:
Right but can we be a bit more specific?
"We support all Miele devices that work with ......"

As in, if I buy a Miele that is 15 years old, it wouldn't work most likely, so what do I need to know if I am standing in the store to know if it would work or not?